docs/styles.css

```
@@ -0,0 +1,302 @@
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  color: var(--ink);
+  background:
+    radial-gradient(circle at top right, rgba(124, 156, 255, 0.18), transparent 30%),
+    radial-gradient(circle at left center, rgba(255, 141, 102, 0.12), transparent 28%),
+    linear-gradient(180deg, #0a0d12 0%, #0f1520 100%);
+  font-family: "Sora", system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Sora" (font-family-name-quotes)

(font-family-name-quotes)